frontend/models/feedback-task-model/record-response/RecordResponse.model.js

```diff
@@ -0,0 +1,17 @@
+import { Model } from "@vuex-orm/core";
+
+class RecordResponse extends Model {
+  static entity = "recordResponses";
+  static primaryKey = ["question_id", "record_id"];
+
+  static fields() {
+    return {
+      id: this.uid(),
+      outputs: this.attr([]),
+      question_id: this.attr(null),
+      record_id: this.attr(null),
```

Comment:
A response is given by a user. So, to support multi-user response, I think `user_id` is a required field here (even if will correspond to the connected user)

Reply:
Ok make sense. I will add the user_id